service.datadog.yaml

```diff
@@ -1,22 +1,188 @@
 ---
-schema-version: v2
+schema-version: v2.1
 dd-service: datadog-agent-nightly
+contacts:
+  - type: slack
+    # #agent-delivery-help
+    contact: https://dd.slack.com/archives/C06PQ7H6PHP
+  - type: email
+    contact: team-agentdelivery@datadoghq.com
+links:
+  - name: datadog-agent
+    provider: github
+    type: repo
+    url: https://github.com/DataDog/datadog-agent
+  - name: internal-chart
+    provider: github
+    type: repo
+    url: https://github.com/DataDog/k8s-datadog-agent-ops/tree/main/k8s/datadog-agent
+  - name: internal-chart-with-operator
+    provider: github
+    type: repo
+    url: https://github.com/DataDog/k8s-datadog-agent-ops/tree/main/k8s/datadog-agent-with-operator
+  - name: internal-image
+    provider: github
+    type: repo
+    url: https://github.com/DataDog/images/tree/master/datadog-agent
+  - name: deployment
+    provider: github
+    type: repo
+    url: https://github.com/DataDog/k8s-datadog-agent-ops
+  - name: Runbook
+    type: doc
+    url: https://datadoghq.atlassian.net/wiki/spaces/agent/pages/986480852/On-Call+Playbooks
+  - name: On Call documentation
+    provider: confluence
+    type: doc
+    url: https://datadoghq.atlassian.net/wiki/spaces/agent/pages/837812317/Combined+Agent+On-call
+  - name: Agent documentation
+    provider: confluence
+    type: doc
+    url: https://datadoghq.atlassian.net/wiki/spaces/agent/pages/2530706500/Agent+Components+in+Kubernetes#Datadog-Cluster-Agent
+tags:
+  - team:container-integrations
+  - service:datadog-agent
+  - app:datadog-agent
 
 extensions:
   datadoghq.com/sdp:
-    workday_team: "Agent Build And Releases"
+    workday_team: "Agent Delivery"
     conductor:
       slack: "datadog-agent-pipelines"
-      schedule: "0 3 * * 1-5"
-      options:
-        disable_bia: true
       targets:
-        - name: "staging"
+        - name: "beta-build-full"
+          # For now, this config is mandatory, even if the target is not a ci_pipeline CNAB object.
+          # Setting this to a placeholder value for now, the intended pipeline that will be triggered will
+          # be the one that is defined in the main .gitlab-ci.yml file.
           ci_pipeline: "//fake_placeholder:fake_placeholder"
           branch: "main"
+          schedule: "10 3 * * SUN-THU"
+          workflows:
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.staging-deploy"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.staging-validate"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.prod-wait-business-hours"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.prod-deploy"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.prod-validate"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.publish-image-confirmation"
```

Comment:
Do we plan on moving the workflows themselves to the `datadog-agent` repository at some point?

Reply:
Also: in the current state, does this workflow still create a `k8s-datadog-agent-ops` pipeline, or does it directly create a `datadog-agent` pipeline?

Reply:
I *think* the workflow are not meant to be moved but I'd rather get a confirmation from @sblumenthal 

This conductor target does not create a pipeline on `k8s-datadog-agent-ops`, but the conductor run on that repo still creates a downstream pipeline in `datadog-agent` and will keep doing so until we completely move conductor from their repo to here.